docs/github-actions.rst

```diff
@@ -36,4 +36,24 @@ Most commonly, you can use it to validate an instance (some data) using a specif
 replacing ``lua-jsonschema`` and the filenames with your implementation and schema of choice.
 For full details on the commands available, see the `CLI documentation <cli>`.
 
+Including Bowtie Output in Workflow summary
+-------------------------------------------
+
+Commands such as bowtie summary, bowtie smoke and bowtie info support outputting in markdown format using the
```

Comment:
Great! Can you add links, e.g.

```suggestion
Commands such as `bowtie summary`, bowtie smoke and bowtie info support outputting in markdown format using the
```

(and same for the rest) -- also should work for the `--format markdown` line, but the syntax for that can be tricky, you might need `--format markdown <bowtie summary --format>`.

Also please use [one sentence per line](https://nick.groenen.me/notes/one-sentence-per-line/) for docs.

Reply:
I wasnt able to get this :

> -- also should work for the --format markdown line, but the syntax for that can be tricky, you might need --format markdown <bowtie summary --format>.

Could you explain a bit more ?